trafficserver-9.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: trafficserver-9
+  version: 9.2.3
+  epoch: 0
+  description: Apache Traffic Server™ is a fast, scalable and extensible HTTP/1.1 and HTTP/2 compliant caching proxy server.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libtool
+      - linux-headers
+      - openssl-dev
+      - pcre-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: b4e3c73354d2a104ed6f00379eeb9f121a85e8bb
+      repository: https://github.com/apache/trafficserver
+      tag: ${{package.version}}
+
+  - runs: |
+      autoreconf -if
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
```

Comment:
Needs an update block and a small test  you can find lot of these in the repo , here is an [example](https://github.com/wolfi-dev/os/blob/main/helm.yaml#L34-L45)

Reply:
@ajayk I updated test, can you take a look?